nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/AccessResource.java

```diff
@@ -170,16 +105,35 @@
     private JwtAuthenticationProvider jwtAuthenticationProvider;
     private JwtService jwtService;
     private OtpService otpService;
-    private OidcService oidcService;
     private KnoxService knoxService;
     private KerberosService kerberosService;
 
-    private SAMLService samlService;
-    private SAMLStateManager samlStateManager;
-    private SAMLCredentialStore samlCredentialStore;
 
-    private IdpUserGroupService idpUserGroupService;
-    private LogoutRequestManager logoutRequestManager;
+//    Subresource locator: obtains the subresource Employee
+//    from the path /employeeinfo/employees/{empid}
+//    @Path("/employees/{empid}")
+//    public Employee getEmployee(@PathParam("empid") String id) {
+//        // Find the Employee based on the id path parameter
+//        Employee emp = ...;
+//        ...
+//        return emp;
+//    }
+
+
+//
+//    @GET
+//    @Consumes(MediaType.WILDCARD)
+//    @Produces(MediaType.WILDCARD)
+//    @Path("oidc/logoutCallback")
+//    @ApiOperation(
+//            value = "Redirect/callback URI for processing the result of the OpenId Connect logout sequence.",
+//            notes = NON_GUARANTEED_ENDPOINT
+//    )
+//    public void oidcLogoutCallback(@Context HttpServletRequest httpServletRequest, @Context HttpServletResponse httpServletResponse) throws Exception {
+//        // only consider user specific access over https
+//
+//    }
+
```

Comment:
Is there a reason for adding this commented section?  It looks like these lines could be removed.